web/src/installers/index.ts

```diff
@@ -419,6 +420,7 @@ export const kurlConfigSchema = {
     additionalNoProxyAddresses: { type: "array", items: { type: "string" }, description: "Addresses that can be reached without a proxy" },
     airgap: { type: "boolean", flag: "airgap", description: "Indicates if this install is an airgap install" },
     hostnameCheck: { type: "string", flag: "hostname-check" , description: "Used as a check during an upgrade to ensure the script will run only on the given hostname" },
+    hostPreflights: { type: "object", description: "Used to add additional host preflight checks."},
```

Comment:
Was this tested?

Reply:
Not sure how to test it... Not really sure what it does but I am open to suggestions.